scripts/setup_host.sh

```diff
@@ -197,6 +202,7 @@ function ubu_thermal_conf() {
 
 function set_host_ui() {
     if [[ $1 == "headless" ]]; then
+        setup_power_button
```

Comment:
do not bind this feature to graphics mode. Add a new option in parse_arg() like "--ignore-pwr-vol-button". What do you think?

Reply:
I checked with Kaushlendra, the owner of Power Management feature. He told me that they are not claiming suspend/resume feature for non-gvtd mode. So need to bind this change with gvtd check.

Reply:
OK, approved